.github/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+closes #
+
+---
+Submission checklist: 
+
+<!-- Remove anything below that is not applicable -->   
+
+#### Layout
+- [ ] Change inspected in the desktop web ui
+- [ ] Change inspected in the mobile web ui
+
+#### Compatibility
+- [ ] Functionality of change validated with a connected multisig
+- [ ] Applicable elements hidden / disabled for watched accounts
+
+#### Maintenance
+- [ ] Tests added 
+- [ ] Tests updated
+- [ ] Issue for test coverage created
+- [ ] Documentation added
+- [ ] Documentation updated
```

Comment:
While it's probably good to have tests etc at the same time, I would say it's too early for addind test at the same time.
with added/updated those are either or afaiu, where I think we should go for all checkboxes checked as a visual sign to be able to merge.